aggregators/will-get-add-to-homescreen-prompt/index.js

```diff
@@ -18,23 +18,48 @@
 'use strict';
 
 const Aggregate = require('../aggregate');
-const AddToHomescreen = require('./add-to-homescreen');
-const WorksOffline = require('./works-offline');
-const MobileFriendly = require('./mobile-friendly');
 
-class WillGetAddToHomescreenPrompt extends Aggregate {
+class AddToHomescreen extends Aggregate {
 
   static get name() {
     return 'Will Get Add to Homescreen Prompt';
   }
 
   static get criteria() {
-    return Object.assign({},
-      WorksOffline.criteria,
-      AddToHomescreen.criteria,
-      MobileFriendly.criteria
-    );
+    const manifestExists = require('../../audits/manifest/exists').name;
+    const manifestBackgroundColor = require('../../audits/manifest/background-color').name;
+    const manifestIcons = require('../../audits/manifest/icons').name;
+    const manifestIcons192 = require('../../audits/manifest/icons-192').name;
+    const manifestShortName = require('../../audits/manifest/short-name').name;
+
+    const criteria = {};
```

Comment:
ah that's cool. i'll hold off for now though. makes things a bit more copypasteable.